flocker/node/testtools.py

```diff
@@ -4,14 +4,26 @@
 Testing utilities for ``flocker.node``.
 """
 
-from unittest import skipIf
+from unittest import skipUnless
 from subprocess import Popen
 
 from ..testtools import loop_until
 
-# This is terible (https://clusterhq.atlassian.net/browse/FLOC-85):
-if_docker_configured = skipIf(Popen([b"docker", b"version"]).wait(),
-                              "Docker must be installed and running.")
+
+def docker_running():
+    """
+    :return: ``True`` if Docker is installed and running, else ``False``.
+
+    XXX This is terrible (https://clusterhq.atlassian.net/browse/FLOC-85)
```

Comment:
It would probably be better to use something like https://github.com/ClusterHQ/flocker/commit/6c0dcb8c980c35f7cfb4989b1dbd82fd14a6c778 since we probably *don't* want to use boot2docker in the tests. (Maybe along with https://github.com/ClusterHQ/flocker/blob/master/flocker/node/_docker.py#L271).